packages/client/core/src/types/cryptid.ts

```diff
@@ -1,8 +1,18 @@
-import { PublicKey, Transaction } from "@solana/web3.js";
+import { Signer, Transaction } from "@solana/web3.js";
 import { CryptidTransaction } from "../lib/CryptidTransaction";
 
 export type ProposalResult = {
   proposeTransaction: Transaction;
-  transactionAccountAddress: PublicKey;
+  transactionAccount: Signer;
```

Comment:
Suggest generalising to array of signers

Reply:
Also suggest to use just a single type { Transaction[], Signer[] } to be future-proof